packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts

```diff
@@ -64,7 +64,30 @@ const wait = (ms = 1000) => {
   });
 }
 
-export class TerraformCloud implements Terraform  {
+function BeaufifyErrors(target: any, propertyKey: string, descriptor: PropertyDescriptor) {
```

Comment:
This should probably be `BeautifyErrors`

Reply:
🙈